### DISCUSSION TITLE
Sample project?

### DISCUSSION THREAD

**Discussion by lonix1** [adamralph/minver#788] [Q&A] [answered]
Does someone have a link to a sample project?

Thank you!

(e.g. I don't understand what I'm supposed to place in the `<Version>` property in my `Foo.csproj`, and whether this lib will overwrite what I put in there. A sample project would easily clear up such confusion.)

**Comment by lonix1** [comment, marked as answer]
Tinkered with it for a while, no need for sample any more. Thanks.

**Reply by adamralph** [reply]
It should be as simple as following https://github.com/adamralph/minver#quick-start

No need to do anything with the `Version` property yourself.

**Comment by lonix1** [comment]
@adamralph I need to congratulate you on creating something so simple and effective. It's far more pleasant than "the other ones" :smirk:

Thank you!

**Reply by adamralph** [reply]
thanks! :slightly_smiling_face:

**Comment by lonix1** [comment]
PS: since you're the resident msbuild / building / versioning expert... 

I'm simplifying a complicated build environment that uses bash scripts. That's why I found MinVer. I'm considering replacing my bash scripts with something simpler.

Given that you prefer simple tools (e.g. MinVer), what do you use for building? msbuild, shell scripts, or maybe something like "cake"?

**Reply by adamralph** [reply]
I happen to have a package designed for exactly that – https://www.nuget.org/packages/Bullseye :slightly_smiling_face: 

Most often used together with https://www.nuget.org/packages/SimpleExec, which takes care of running commands, such as `dotnet build`, etc.

**Reply by lonix1** [reply]
I saw that lib but didn't realise it's a build tool! I'll take a proper look. Thanks.

**Reply by adamralph** [reply]
That's very useful feedback, thank you. I'll see what I can do to make it more obvious.

**Reply by adamralph** [reply]
@lonix1 FYI I recently updated the [Bullseye README](https://github.com/adamralph/bullseye/blob/main/README.md) to make it more obvious that it's intended to be a build tool.
